packages/kamino-sdk/src/whirpools-client/shim/remove-liquidity.ts

```diff
@@ -0,0 +1,127 @@
+// Source: https://raw.githubusercontent.com/orca-so/whirlpool-sdk/main/src/position/quotes/remove-liquidity.ts
```

Comment:
I copied this from the orca sdk to add a rounding flag. Open to better suggestions..